kitsune/search/views.py

```diff
@@ -346,21 +207,17 @@ def simple_search(request, template=None):
         t = 'search/mobile/down.html' if request.MOBILE else 'search/down.html'
         return render(request, t, {'q': cleaned['q']}, status=503)
 
-    items = [(k, v) for k in search_form.fields for
-             v in r.getlist(k) if v and k != 'a']
-    items.append(('a', '2'))
-
     fallback_results = None
     if num_results == 0:
         fallback_results = _fallback_results(language, cleaned['product'])
 
     product = Product.objects.filter(slug__in=cleaned['product'])
     if product:
-        product_titles = [_(p.title, 'DB: products.Product.title')
-                          for p in product]
+        product_titles = [_(p.title, 'DB: products.Product.title') for p in product]
     else:
         product_titles = [_('All Products')]
 
+    # FIXME: This is probably bad l10n.
```

Comment:
Atom has a plugin for that.

It's an interesting idea, but I'm not sure how it helps us. For things that need to get done, I'll spin off new bugs. For everything else, I'm adding notes for someone in the future who's spelunking in that code area so they are better informed about the code. They're not necessarily things I think need to get fixed.